test/RiskManagerV1.test.js

```diff
@@ -35,6 +35,10 @@ describe("RiskManagerV1", () => {
     signerBondsSwapStrategy = await SignerBondsSwapStrategy.deploy()
     await signerBondsSwapStrategy.deployed()
 
+    // TODO: Another type of strategy could be used
```

Comment:
I think it's not relevant in the context of this test. The most important thing we need here is another address.

Reply:
Fixed in d295b8363bac9bc886cb50c33525c461ac2c3721.